README.md

```diff
@@ -86,20 +86,25 @@ Gentofte - ID: 157
 Gladsaxe - ID: 159
 Glostrup - ID: 161
 Greve - ID: 253
+Halsnæs - ID: 260
 Helsingør - ID: 217
 Herlev - ID: 163
 Hillerød - ID: 219
 Hvidovre - ID: 167
 Høje-Taastrup - ID: 169
 Køge - ID: 259
 Lejre - ID: 350
+Lyngby-Taarbæk - ID: 173
 Rebild - ID: 840
 Ringkøbing-Skjern - ID: 760
 Roskilde - ID: 265
 Rudersdal - ID: 230
 Solrød - ID: 269
 Stevns - ID: 336
 Svendborg - ID: 479
+Sweco Demo - ID: 10210
+Sweco Test B - ID: 16161
+Sweco Test T - ID: 20161
```

Comment:
These where left out as they are useless, and give a false total of supported cities.
I would suggest removing them.

Are they added by accident, or deliberately ?

Reply:
I just took all the id and "municipalities / names" without checking them.
I remove the 3 lines of test / demo